src/Microsoft.OpenApi.Readers/Exceptions/OpenApiReaderException.cs

```diff
@@ -22,6 +24,17 @@ public class OpenApiReaderException : Exception
         /// <param name="message">Plain text error message for this exception.</param>
         public OpenApiReaderException(string message) : base(message) { }
 
+        /// <summary>
+        /// Initializes the <see cref="OpenApiReaderException"/> class with a message and line, column location of error.
+        /// </summary>
+        /// <param name="message">Plain text error message for this exception.</param>
+        /// <param name="node">Parsing node where error occured</param>
+        public OpenApiReaderException(string message, YamlNode node) : base(message)
+        {
+            Pointer = $"#line={node.Start.Line}";  // This only includes line because using a char range causes tests to break due to CR/LF & LF differences
+                                                   // See https://tools.ietf.org/html/rfc5147 for syntax
```

Comment:
Can comments go on top of the command line? This is inconsistent with the rest of the project.